plugin.js

```diff
@@ -177,6 +177,8 @@ const buildAuthenticatedRouter = (
     ...sessionOptions,
     secret: auth.cookiePassword,
     name: auth.cookieName || 'adminbro',
+    resave: true,
+    saveUninitialized: true
```

Comment:
these are the options that the user may want to change. By hardcoding them in the library you're disabling the possibility to customize that.